index.js

```diff
@@ -183,12 +184,14 @@ class Ky {
 					}
 				}
 
-				if (!response.ok) {
+				if (!response.ok && (isRetry || (!isRetry && throwHttpErrors))) {
```

Comment:
It works as is but using "this.throwHttpErrors" would be more in-sync with the rest of the code.

```suggestion
				if (!response.ok && (isRetry || (!isRetry && this.throwHttpErrors))) {
```

Reply:
Saying `isRetry || (!isRetry && throwHttpErrors)` is the same as just saying `isRetry || throwHttpErrors`